Makefile

```diff
@@ -20,9 +20,9 @@ test: build
 host-build:
 	./checks "./*.go ./libpkt/ ./ofctrl/ ./ovsdbDriver/ ./ovsSwitch/ ./pqueue/ ./rpcHub/"
 	go get github.com/tools/godep
-	godep go install ./ ./ofctrl
+	go install ./ ./ofctrl
 
 host-test:
-	sudo -E PATH=$(PATH) /opt/gopath/bin/godep go test -v ./
-	sudo -E PATH=$(PATH) /opt/gopath/bin/godep go test -v ./ofctrl
-	sudo -E PATH=$(PATH) /opt/gopath/bin/godep go test -v ./pqueue
+	PATH=${PATH} sudo -E /usr/local/go/bin/go test -v ./
```

Comment:
If the box isn't being updated, do we still need the `host-test` path changes in `Makefile`?  IIRC the old box had Go installed under the `/opt` path